### PR TITLE
[TRANSLATION] Translate MITLA.md

### PR DESCRIPTION
**Pull request by tangfqj**
I've translated MITLA.md, please check whether there are anything improper or wrong.
Hope it does some help to you!

### CODE REVIEW

File: docs/数学基础/MITLA.en.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# MIT18.06: Linear Algebra
+
+## Descriptions
+
+- Offered by: MIT
+- Prerequisites: English
+- Programming languages: None
+- Difficulty: 🌟🌟🌟
+- Class Hour: Varies according to the learner
```
Comment (PKUFlyingPig) [inline docs/数学基础/MITLA.en.md:9]: Varying from person to person

File: docs/数学基础/MITLA.en.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Gilbert Strang, a great mathematician, still insists on teaching in his old age. His classic text book [Introduction to Linear Algebra](https://math.mit.edu/~gs/linearalgebra/)has been adopted as an official textbook by Tsinghua University. After reading the PDF version, I felt deeply guilty and spent more than 200 yuan to purchase a genuine version in English as collection. The cover of this book is attached below. If you can fully understand the mathematical meaning of the cover picture, then your understanding of linear algebra will definitely reach a new height.
```
Comment (PKUFlyingPig) [inline docs/数学基础/MITLA.en.md:11]: a great mathematician at MIT, still insists on teaching in his eighties.

File: docs/数学基础/MITLA.en.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+![image](https://math.mit.edu/~gs/linearalgebra/linearalgebra5_Front.jpg)
+
+If you combine the course with the youtube blogger **3Blue1Brown**'s[The Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab)videos, you may learn better.
```
Comment (PKUFlyingPig) [inline docs/数学基础/MITLA.en.md:15]: In addition to the course materials, the famous Youtuber **3Blue1Brown**'s video series [The Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab) are also great learning resources.

**Review by PKUFlyingPig — CHANGES_REQUESTED** [review]
感谢贡献，一些语句的细节做了调整。

**Comment by tangfqj** [comment]
好的，我按照你的调整修改一下。
cs自学指南真的太有用了，很高兴能为这个项目做点贡献。

**Pull request merged**